pvmismatch/contrib/ModuleMismatchSimulator.py

```diff
@@ -0,0 +1,510 @@
+'''
```

Comment:
Also, can you change the name to follow [pep8 package and module naming conventions](https://www.python.org/dev/peps/pep-0008/#package-and-module-names)? thanks!